frontend/packages/dev-console/src/components/add/DeveloperFeaturesGettingStartedCard.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { FlagIcon } from '@patternfly/react-icons';
+
+import { ALL_NAMESPACES_KEY, useActiveNamespace, useOpenshiftVersion } from '@console/shared/src';
+import { GettingStartedCard } from '@console/shared/src/components/getting-started';
+import { GettingStartedLink } from '@console/shared/src/components/getting-started/types';
+
+export const DeveloperFeaturesGettingStartedCard = () => {
+  const { t } = useTranslation();
+  const [activeNamespace] = useActiveNamespace();
+  // Show only major and minor version.
+  const version = (useOpenshiftVersion() || '')
```

Comment:
How about 
```suggestion
  const version = (useOpenshiftVersion() || '4.x')
```

Reply:
I would like to stay with an empty string and show only "What's new in OpenShift" in that case.

Reply:
@jerolimov I think it's good to be consistent as here we have `4.x` https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/guided-tour/GuidedTourText.tsx#L16

either way is fine but shouldn't it be same?